assets/src/edit-story/components/moveable/cropStyle.js

```diff
@@ -139,7 +139,7 @@ export const GlobalStyle = createGlobalStyle`
 	}
 
 	.crop-moveable .moveable-direction.moveable-line {
-		background: #47a0f4 !important;
+		background: #1A73E8 !important;
```

Comment:
No way to bring themes here? If not, can we at least put in a comment `== theme.x.y.z`?